backend/docker-bake.hcl

```diff
@@ -0,0 +1,41 @@
+# This is a docker bake file in HCL syntax.
```

Comment:
These files always confuse me so bad lol but I guess we need them for the caching stuff?